src/main/java/com/stripe/model/Invoice.java

```diff
@@ -152,10 +154,6 @@
   @SerializedName("ending_balance")
   Long endingBalance;
 
-  /** The time that the invoice draft was finalized. */
-  @SerializedName("finalized_at")
```

Comment:
Will use `Status_transitions` instead.
This is breaking, but I will have to document them again in the diff.